base/src/commonTest/kotlin/text/DateFormatterTest.kt

```diff
@@ -60,10 +65,36 @@ class DateFormatterTest {
         assertEquals(8, date.day)
     }
 
+    @Test
+    fun testDateFormat() {
+        val usFormatter = DateFormatter.dateFormat(DateFormatStyle.Medium, TimeZone.utc, UnitedStatesLocale)
+        val frFormatter = DateFormatter.dateFormat(DateFormatStyle.Medium, TimeZone.utc, FranceLocale)
+
+        assertEquals("Jan 8, 1988", usFormatter.format(January81988))
+        assertEquals("8 janv. 1988", frFormatter.format(January81988))
+    }
+
+    @Test
+    fun testDateFormatWithoutYear() {
+        val usFormatter = DateFormatter.dateFormat(DateFormatStyle.Medium, true, TimeZone.utc, UnitedStatesLocale)
```

Comment:
It's a bit tricky and not clear that this formatter will return result without year. But it's only option in current setup.
The better way is to use date components to be passed to formatter to build the date.
For example: `format([.day, .month], .utc, .en_US)` then it's clear what to expect.

Reply:
Like you said, its not setup like this atm. If we want/need this, I suggest someone writes an issue for it :)